src/Accordion.php

```diff
@@ -88,12 +91,17 @@ public function renderItems(): string
     {
         $items = [];
         $index = 0;
+        $expanded = array_search(true, array_column(ArrayHelper::toArray($this->items), 'expand'));
```

Comment:
What's the goal of `ArrayHelper::toArray()` here? Items should always be an array, right?

Reply:
@hoaaah I think this was your work on https://github.com/yiisoft/yii2-bootstrap4/commit/21b7ba166fe47e609d4093bd89da898c66fb627b. Any thoughts on this?